src/module/Ui/templates/authentication/login.twig

```diff
@@ -14,13 +14,13 @@
 <div class="form-group">
     <div class="input-group">
         <span class="input-group-addon"><i class="fa fa-user"></i></span>
-        {{ formElement(form.get('email')) }}
+        {{ formElement(form.get('email').setAttribute('placeholder', 'Email'|trans)) }}
```

Comment:
That's where they were, but they didn't get translated. Is there a way to apply translation immediately in the class?